layout/minimal.liquid

```diff
@@ -25,9 +25,14 @@
     {% render 'root-styles', settings: settings %}
 
     {% render 'page-metadata',
+      canonical_url: canonical_url,
       page_title: page_title,
       page_description: page_description,
-      shop: shop
+      page_meta_image_url: page_meta_image_url,
+      page_meta_image_alt: page_meta_image_alt,
+      product: product,
+      shop: shop,
+      locale: locale
     %}
```

Comment:
Here we can isolate of rendering `page-metadata` snippet from the Preview app at all

```
{%- unless section_preview -%}
    {% render 'page-metadata',
          canonical_url: canonical_url,
          page_title: page_title,
          page_description: page_description,
          shop: shop
          page_meta_image_url: page_meta_image_url,
          page_meta_image_alt: page_meta_image_alt,
          product: product,
          shop: shop,
          locale: locale
    %}
{%- endunless -%}
```

Reply:
sounds good, will update